src/lib.rs

```diff
@@ -289,7 +317,9 @@ mod tests {
         flags.push(RandomXFlag::FlagDefault);
         let cache = RandomXCache::new(flags.clone(), key).unwrap();
         let dataset = RandomXDataset::new(flags.clone(), &cache, 0).unwrap();
-        let memory = dataset.get_data();
+        let memory_result = dataset.get_data();
+        assert_eq!(memory_result.is_ok(), true);
+        let memory = memory_result.unwrap();
```

Comment:
`let memory = dataset.get_data().expect("...");` does this in one line.

Reply:
updated